code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -1,12 +1,36 @@
 /datum/emote/living/carbon/human
 	mob_type_allowed_typecache = list(/mob/living/carbon/human)
 
+#define CRY_DURATION 12.8 SECONDS
+
 /datum/emote/living/carbon/human/cry
 	key = "cry"
 	key_third_person = "cries"
 	message = "cries."
 	message_mime = "sobs silently."
 	emote_type = EMOTE_AUDIBLE | EMOTE_VISIBLE
+	/// Timer for the visual effect to wear off
+	var/visual_timer = TIMER_ID_NULL
+
+/datum/emote/living/carbon/human/cry/run_emote(mob/user, params, type_override, intentional)
+	. = ..()
+	if(. && isliving(user))
+		var/mob/living/living_user = user
+		ADD_TRAIT(living_user, TRAIT_CRYING, "[type]")
+		living_user.update_body()
+
+		// Use a timer to remove the effect after the defined duration has passed
+		var/list/key_emotes = GLOB.emote_list["cry"]
+		for(var/datum/emote/living/carbon/human/cry/living_emote in key_emotes)
+			// The existing timer restarts if it's already running
+			visual_timer = addtimer(CALLBACK(living_emote, .proc/end_visual, living_user), CRY_DURATION, TIMER_UNIQUE | TIMER_OVERRIDE)
```

Comment:
doesnt this need `TIMER_STOPPABLE` to return an ID?

Reply:
It seems that without that flag it returns the value of TIMER_ID_NULL. With that flag it uses some other number. But it turns out that the timer restarts just fine without needing to track the timer ID. Guess that makes sense because I only assigned it but never used it.